doctor/forms.py

```diff
@@ -30,12 +30,12 @@ def clean_file(self):
         return file
 
     def prep_file(self):
-        fp = tempfile.NamedTemporaryFile(
-            delete=False, suffix=f'.{self.cleaned_data["extension"]}'
-        )
-        self.cleaned_data["tmp_dir"] = tempfile.TemporaryDirectory()
-        self.cleaned_data["fp"] = fp.name
-        self.temp_save_file(fp.name)
+        with tempfile.NamedTemporaryFile(
+            delete=False, suffix=f".{self.cleaned_data['extension']}"
+        ) as fp:
```

Comment:
```
doctor/forms.py:33:14: SIM115 Use a context manager for opening files
   |
32 |     def prep_file(self):
33 |         fp = tempfile.NamedTemporaryFile(
   |              ^^^^^^^^^^^^^^^^^^^^^^^^^^^ SIM115
34 |             delete=False, suffix=f".{self.cleaned_data['extension']}"
35 |         )
   |
```

https://docs.astral.sh/ruff/rules/open-file-with-context-handler/

Had a lot of these errors to fix in this repository